centaur/src/main/resources/standardTestCases/wdl_draft3/draft3_nio_file/draft3_nio_file.wdl

```diff
@@ -0,0 +1,118 @@
+version 1.0
+
+workflow draft3_nio_file {
```

Comment:
name conflicts with version string above

Reply:
also it seems like nearly every *draft3* `.test` submits a `draft-3` wdl